blob/blob.go

```diff
@@ -904,9 +909,26 @@ type SignedURLOptions struct {
 	// Expiry sets how long the returned URL is valid for.
 	// Defaults to DefaultSignedURLExpiry.
 	Expiry time.Duration
+
 	// Method is the HTTP method that can be used on the URL; one of "GET", "PUT",
 	// or "DELETE". Defaults to "GET".
 	Method string
+
+	// ContentType specifies the Content-Type HTTP header the user agent is
+	// permitted to use in the PUT request. It must match exactly. See
+	// EnforceAbsentContentType for behavior when ContentType is the empty string.
+	//
+	// Must be empty for non-PUT requests.
```

Comment:
Maybe add something here to indicate that this is not supported by all drivers?

Reply:
(i.e., IIUC azureblob will return an Unimplemented error if you put something here).

Reply:
Added docs here and in driver to that effect.